### PR TITLE
CPM-1884

### PR DESCRIPTION
**Pull request by joeyrogues**
https://contractlive.atlassian.net/browse/CPM-1884

### CODE REVIEW

File: addon/instance-initializers/browser-update.js [hunk 1]
```diff
@@ -1,11 +1,5 @@
 export function initialize(instance) {
```
Comment (joeyrogues) [inline addon/instance-initializers/browser-update.js]: I should remove it completly

File: index.js [hunk 2]
```diff
@@ -2,5 +2,9 @@
 'use strict';
 
 module.exports = {
-  name: 'ember-cli-browser-update'
+  name: 'ember-cli-browser-update',
+
+  included(app) {
+    app.import('vendor/browser-update/update.js', { prepend: true });
```
Comment (joeyrogues) [inline index.js]: Take the script that what previously injected directly in the addon.
Giving us more control over what's injected in our app BTW :package:

File: addon/services/browser-update.js [hunk 3]
```diff
@@ -2,42 +2,8 @@ import Ember from 'ember';
...
-    }, delay);
-  },
-
-  _config: Ember.computed(function () {
-    var config = this.get('config');
-    // send config to browser-update script as global variable
-    config.vs = config.vs || {i: 9, f: 2, o: 9.63, s: 2, c: 10};
-    config.onshow = this.show.bind(this);
-    config.onclick = this.click.bind(this);
-    config.onclose = this.close.bind(this);
```
Comment (ctjhoa) [inline addon/services/browser-update.js]: Du coup tu perds tous ces hooks

Comment (joeyrogues) [reply addon/services/browser-update.js]: Vu à l'oral => oui

**Pull request closed without merging**